charts/tempo-distributed/templates/query/configmap-query.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "tempo.queryFullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "tempo.queryLabels" . | nindent 4 }}
+data:
+  tempo-query.yaml: |
+    backend: localhost:{{ .Values.conf.server.listenPort }}
```

Comment:
This can be moved to the general config file within the root of templates. This file can be removed afterwards.